dagger-core/src/main/java/io/odpf/dagger/core/metrics/reporters/statsd/measurement/Counter.java

```diff
@@ -0,0 +1,15 @@
+package io.odpf.dagger.core.metrics.reporters.statsd.measurement;
+
+import io.odpf.dagger.common.metrics.aspects.Aspects;
+
+import java.io.Serializable;
+
+public interface Counter extends Serializable {
```

Comment:
It seems to me an over-design.. but debatable. I see only one implementation, i don't think we will have more implementation of this. it seems like we are wrapping statsDReporter method with another method. I am not sure how this is useful.

Did we discuss all these interfaces before implementing?